src/modules/commander/Commander.cpp

```diff
@@ -1531,11 +1531,15 @@ Commander::handle_command(const vehicle_command_s &cmd)
 
 				} else if (((int)(cmd.param1)) == 2) {
 					answer_command(cmd, vehicle_command_s::VEHICLE_CMD_RESULT_ACCEPTED);
-					_worker_thread.startTask(WorkerThread::Request::ParamResetAll);
+					_worker_thread.startTask(WorkerThread::Request::ParamResetAllConfig);
```

Comment:
I can see this go either way: update the existing mavlink description for this, or keep the behavior and use param1==4 for ParamResetAllConfig. Let's see what response you get on the mavlink change.

Reply:
Yeah, let's see on mavlink: https://github.com/mavlink/mavlink/pull/1826

Reply:
@bkueng This was merged on mavlink